packages/dotcom/.changeset/config.json

```diff
@@ -7,5 +7,5 @@
   "access": "restricted",
   "baseBranch": "main",
   "updateInternalDependencies": "patch",
-  "ignore": []
+  "ignore": ["@sdc/modules","@sdc/server","@sdc/shared"]
```

Comment:
without these exclusions we get a menu listing the yarn workspaces:
<img width="466" alt="Screenshot 2023-11-23 at 13 21 32" src="https://github.com/guardian/support-dotcom-components/assets/1513454/1b30be8b-b0fd-4c3c-9945-aa6a622a16bb">